api/handlers/app_test.go

```diff
@@ -525,6 +530,133 @@ var _ = Describe("App", func() {
 		})
 	})
 
+	Describe("GET /v3/apps/{guid}/manifest", func() {
+		var dropletRecord repositories.DropletRecord
```

Comment:
How about making `dropletRecord` local to the `BeforeEach` below (i.e. do not `var` it here)?